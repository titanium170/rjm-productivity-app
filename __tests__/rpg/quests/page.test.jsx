import { render } from '@testing-library/react';
import Page from '@/app/rpg/quests/page';
 
describe('Quests page', () => {
  it('renders correctly', async () => {
    const { container } = render(await Page());
    expect(container).toMatchSnapshot();
  })
});