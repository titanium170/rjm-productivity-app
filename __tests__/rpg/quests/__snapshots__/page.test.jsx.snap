// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quests page renders correctly 1`] = `
<div>
  <div
    class="bg-panel-texture bg-cover bg-center border border-opacity-40 border-white rounded-sm shadow-md shadow-slate-950/100 w-96"
  >
    <div
      class="p-8 shadow-inner bg-sky-950 bg-opacity-30 backdrop-blur-[1px] undefined"
    >
      <header
        class="border border-white border-opacity-30 mx-auto text-center w-fit px-14 py-1 relative after:content-[''] after:absolute after:right-[-1px] after:left-[-1px] after:top-[2px] after:bottom-[-3px] after:border after:border-white after:border-opacity-5"
      >
        <p
          class="z-10 text-3xl font-mono tracking-widest uppercase rounded-full w-fit pb-1 shadow-[0px_0px_20px_17px_rgb(80,181,253,0.1),inset_0px_0px_0px_20px_rgb(80,181,253,0.1)]"
          style="text-shadow: 0px 1px 4px white, 0px 3px 0px rgb(85, 138, 187, 0.5);"
        >
          Quest info
        </p>
      </header>
      <dialog>
        <h2>
          Add a new quest
        </h2>
        <form>
          <label
            for="quest-title"
          >
            Quest title
          </label>
          <input
            id="quest-title"
            name="questTitle"
            type="text"
          />
          <label
            for="quest-objective"
          >
            Quest objective
          </label>
          <input
            id="quest-objective"
            name="questObj"
            type="text"
          />
          <button>
            Add quest
          </button>
        </form>
      </dialog>
      <button>
        Add Quest
      </button>
      <ul
        class="p-6"
      >
        <li
          class="pb-2"
        >
          <h3>
            <p
              class="font-light !text-base text-xs rounded-full w-fit font-extralight pb-2 shadow-[0px_0px_20px_17px_rgb(80,181,253,0.05),inset_0px_0px_0px_20px_rgb(80,181,253,0.05)]"
              style="text-shadow: 0px 1px 4px white, 0px 3px 0px rgb(85, 138, 187, 0.3);"
            >
              <strong>
                Boar problem
              </strong>
            </p>
          </h3>
          <p
            class="text-xs indent-2 text-xs rounded-full w-fit font-extralight pb-2 shadow-[0px_0px_20px_17px_rgb(80,181,253,0.05),inset_0px_0px_0px_20px_rgb(80,181,253,0.05)]"
            style="text-shadow: 0px 1px 4px white, 0px 3px 0px rgb(85, 138, 187, 0.3);"
          >
            Kill 10 boars
          </p>
        </li>
        <li
          class="pb-2"
        >
          <h3>
            <p
              class="font-light !text-base text-xs rounded-full w-fit font-extralight pb-2 shadow-[0px_0px_20px_17px_rgb(80,181,253,0.05),inset_0px_0px_0px_20px_rgb(80,181,253,0.05)]"
              style="text-shadow: 0px 1px 4px white, 0px 3px 0px rgb(85, 138, 187, 0.3);"
            >
              <strong>
                Tuskers
              </strong>
            </p>
          </h3>
          <p
            class="text-xs indent-2 text-xs rounded-full w-fit font-extralight pb-2 shadow-[0px_0px_20px_17px_rgb(80,181,253,0.05),inset_0px_0px_0px_20px_rgb(80,181,253,0.05)]"
            style="text-shadow: 0px 1px 4px white, 0px 3px 0px rgb(85, 138, 187, 0.3);"
          >
            Collect 5 tusks
          </p>
        </li>
        <li
          class="pb-2"
        >
          <h3>
            <p
              class="font-light !text-base text-xs rounded-full w-fit font-extralight pb-2 shadow-[0px_0px_20px_17px_rgb(80,181,253,0.05),inset_0px_0px_0px_20px_rgb(80,181,253,0.05)]"
              style="text-shadow: 0px 1px 4px white, 0px 3px 0px rgb(85, 138, 187, 0.3);"
            >
              <strong>
                Mithrandir
              </strong>
            </p>
          </h3>
          <p
            class="text-xs indent-2 text-xs rounded-full w-fit font-extralight pb-2 shadow-[0px_0px_20px_17px_rgb(80,181,253,0.05),inset_0px_0px_0px_20px_rgb(80,181,253,0.05)]"
            style="text-shadow: 0px 1px 4px white, 0px 3px 0px rgb(85, 138, 187, 0.3);"
          >
            Kill gandalf
          </p>
        </li>
        <li
          class="pb-2"
        >
          <h3>
            <p
              class="font-light !text-base text-xs rounded-full w-fit font-extralight pb-2 shadow-[0px_0px_20px_17px_rgb(80,181,253,0.05),inset_0px_0px_0px_20px_rgb(80,181,253,0.05)]"
              style="text-shadow: 0px 1px 4px white, 0px 3px 0px rgb(85, 138, 187, 0.3);"
            >
              <strong>
                Snazzy
              </strong>
            </p>
          </h3>
          <p
            class="text-xs indent-2 text-xs rounded-full w-fit font-extralight pb-2 shadow-[0px_0px_20px_17px_rgb(80,181,253,0.05),inset_0px_0px_0px_20px_rgb(80,181,253,0.05)]"
            style="text-shadow: 0px 1px 4px white, 0px 3px 0px rgb(85, 138, 187, 0.3);"
          >
            Create nice UI
          </p>
        </li>
      </ul>
    </div>
  </div>
</div>
`;
